src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -214,7 +215,7 @@ public override void OnStopActivity(Activity activity, object payload)
                 }
             }
 
-            if (activity.OperationName.Equals(ActivityNameByHttpInListener, StringComparison.Ordinal))
+            if (activity.CheckFirstTag("IsCreatedByInstrumentation", out var tagValue) && ReferenceEquals(tagValue, bool.TrueString))
```

Comment:
should we just remove this tag here? (if you set a tag value to null, that would be as good as removing it). Otherwise an exporter sees this as just any other tag. We don't want to leak the implementation details there.

Reply:
Fixed this.